code/modules/holiday/holidays.dm

```diff
@@ -202,6 +205,7 @@
 	name = "UFO Day"
 	begin_day = 2
 	begin_month = JULY
+	drone_hat = /obj/item/clothing/mask/facehugger/dead 
```

Comment:
🔫👮👮👮👮👮👮👮👮
WEE WOO WEE WOO
YOU ARE IN VIOLATION OF DRONE LAW 2 STEP AWAY FROM THE BEING